packages/cluster-ui/src/statementDetails/statementDetails.tsx

```diff
@@ -850,3 +851,595 @@ export class StatementDetails extends React.Component<
+                {
+                  name: "Overhead",
+                  value: stats.overhead_lat,
+                  bar: overheadBarChart,
+                },
+                {
+                  name: "Overall",
+                  summary: true,
+                  value: stats.service_lat,
+                  bar: overallBarChart,
+                },
+              ]}
+            />
+          </SummaryCard>
+          <SummaryCard>
+            <h2
+              className={classNames(
+                cx("base-heading"),
+                summaryCardStylesCx("summary--card__title"),
+              )}
+            >
+              Other Execution Statistics
+            </h2>
+            <NumericStatTable
+              title="Stat"
+              measure="Quantity"
+              count={count}
+              format={d3Format(".2f")}
+              rows={[
+                {
+                  name: "Rows Read",
+                  value: stats.rows_read,
+                  bar: genericBarChart(stats.rows_read, stats.count),
+                },
+                {
+                  name: "Disk Bytes Read",
+                  value: stats.bytes_read,
+                  bar: genericBarChart(stats.bytes_read, stats.count, Bytes),
+                  format: Bytes,
+                },
+                {
+                  name: "Network Bytes Sent",
+                  value: stats.bytes_sent_over_network,
+                  bar: genericBarChart(
+                    stats.bytes_sent_over_network,
+                    stats.count,
+                    Bytes,
+                  ),
+                  format: Bytes,
+                },
+              ].filter(function(r) {
+                if (
+                  r.name === "Network Bytes Sent" &&
+                  r.value &&
+                  r.value.mean === 0
+                ) {
+                  // Omit if empty.
+                  return false;
+                }
+                return r.value;
+              })}
+            />
+          </SummaryCard>
+          <SummaryCard>
+            <h2
+              className={classNames(
+                cx("base-heading"),
+                summaryCardStylesCx("summary--card__title"),
+              )}
+            >
+              Stats By Node
+              <div className={cx("numeric-stats-table__tooltip")}>
+                <Tooltip text="Execution statistics for this statement per gateway node.">
+                  <div
+                    className={cx("numeric-stats-table__tooltip-hover-area")}
+                  >
+                    <div className={cx("numeric-stats-table__info-icon")}>
+                      i
+                    </div>
+                  </div>
+                </Tooltip>
+              </div>
+            </h2>
+            <StatementsSortedTable
+              className={cx("statements-table")}
+              data={statsByNode}
+              columns={makeNodesColumns_20_2(statsByNode, this.props.nodeNames)}
```

Comment:
`...makeNodesColumns_20_2...`
this is the only difference between `StatementDetails` and `StatementDetails_20_2` components